src/components/common/TmBalance.vue

```diff
@@ -238,10 +238,9 @@ export default {
   },
   data() {
     return {
-      overview: {
-        rewards: []
-      },
+      overview: {},
       sentToGA: false,
+      rewardsSentToGA: false,
```

Comment:
what's this?

Reply:
a checker to see if `totalRewards` were already sent to GA